### PR TITLE
Nginx example

### PR DESCRIPTION
**Pull request by avestuk**
Add nginx kubernetes example

### CODE REVIEW

**Comment by avestuk** [comment]
@actionbuddha This PR also now contains an update for the CSI deployment README.

File: k8s/examples/nginx/README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# Nginx on Kubernetes with StorageOS persistent Storage
+
+This example shows an example of how to deploy Nginx on Kubernetes, in order to
+exfiltrate files that have been written to a StorageOS persistent volume. The
+files create a stateful set that can be used *AFTER* a StorageOS cluster has
+been created. For more information on how to install a StorageOS cluster please see
+[the StorageOS documentation](https://docs.storageos.com/docs/introduction/quickstart)
+for more information.
```
Comment (Arau) [inline k8s/examples/nginx/README.md]: For more information appears twice in the same sentence. With one I think is enough.

**Review by Arau — COMMENTED** [review]
One wording change and good to go. 

This is a great example! Thank you!

**Comment by avestuk** [comment]
Merging now that Ferans comment has been resolved.

**Pull request merged**